### ISSUE TITLE
Broken Link in Security Pod Security Standard

### ISSUE THREAD

**Issue by niranjandarshann** [kubernetes/website#40867]
**This is a Bug Report**

Problem:
Link for `This is Linux only policy`  in  https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted   is broken.
 Various controls having Policy reffering the same link  are:
```
- Privilege Escalation (v1.8+)
- Seccomp (v1.19+)
- Capabilities (v1.22+)
```
broken  

Proposed Solution:
Should be updated with correct link 

Page to Update:
https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/security/pod-security-standards.md

**Comment by dipesh-rawat** [comment]
It appears that the changes in #35772 have inadvertently resulted in the occurrence of the mentioned broken links.

**Comment by niranjandarshann** [comment]
> 

yes @dipesh-rawat  it seems that inadvertently anchor tag this 
`<p>Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed. <em>**<a href="#policies-specific-to-linux">This is Linux only policy</a>** in v1.25+ <code>(spec.os.name != windows)</code></em></p>` 
and mentioned link in the issue is refering to the `policies-specific-to-linux` topic which is absent in the whole page thats why it result in the broken link.

predicted correct link : https://kubernetes.io/docs/concepts/security/pod-security-standards/#os-specific-policy-controls.

I would like to assign it, if this link will be suitable to fix the issue.

**Comment by tengqm** [comment]
Change the target to `(#os-specific-policy-controls)` or remove the link should be fine.
/triage accepted
/kind bug

**Comment by niranjandarshann** [comment]
closing this issue as it is resolved in  PR #40885 
/close
